package.json

```diff
@@ -37,6 +43,7 @@
     "eslint-config-digitalbazaar": "^3.0.0",
     "eslint-plugin-jsdoc": "^39.3.2",
     "eslint-plugin-unicorn": "^42.0.0",
+    "esm": "^3.2.25",
```

Comment:
I think other DCC ported-over libraries include `esm` solely for mocha tests. But it doesn't look like it's being used here (in the test script)